lib/seagullService.js

```diff
@@ -92,17 +90,18 @@ function createServer(serverConfig, crudHandler, userApiClient, gatekeeperClient
     seagullApi.close();
     app.removeAllListeners();
   });
-  return app;
+  return [app, seagullApi];
```

Comment:
This change is a code smell. You are changing the signature to return 2 different objects, only because the status function is defined in `seagullApi`. I'd revert this change and define the function elsewhere. This should be ok, because it only returns an empty value and doesn't depend on any variables defined in `seagullApi`.